docs/tools/api.rst

```diff
@@ -1,48 +1,44 @@
-Aiven API overview
-==================
+Aiven API
+=========
 
-The Aiven API allows you to perform any tasks on Aiven from a client of your choice.
+Use the Aiven API to programmatically perform any task that you can do through the web interface. This is an ideal way to automate tasks involving Aiven at every stage of your workflow.
 
-API quickstart
---------------
+Common use cases for the Aiven API:
+
+* Use with CI (Continuous Integration) to spin up data platforms on Aiven for use during test runs.
+
+* Integrate with other parts of your existing automation setup to achieve complex tasks.
 
-* Try `Aiven on Postman <https://www.postman.com/aiven-apis/workspace/aiven/documentation/21112408-1f6306ef-982e-49f8-bdae-4d9fdadbd6cd>`_ and make your first calls to our API.
+* Deploy and tear down development or demo platforms on a schedule.
 
-* You will need an authentication token from the `profile section of your Aiven console <https://console.aiven.io/profile/auth>`_.
+We make the API available to all Aiven users. It is also the engine behind our web console, so you should find that all operations are also available via API.
```

Comment:
```suggestion
We make the API available to all Aiven users. It is also the engine behind the Aiven Console, so you should find that all operations are also available through the API.
```

Reply:
Assuming this is the Aiven Console you're talking about, Slab says we need to word it like this.